python/ynab_client.py

```diff
@@ -65,6 +67,3 @@ def findPreviousTransaction(payee_name):
         .filter(Transaction.imported_payee==payee_name)\
         .order_by(Transaction.date.desc())\
         .first()
-
-# Sync with YNAB
-sync()
```

Comment:
Hmmm it's not really a good place for it—`ynab_client` is a library, it shouldn't be doing any processing on import.

I'll think about how to do this.